Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -8326,24 +8328,38 @@ def create_outputs(whois_result, domain, query=None):
         contacts = whois_result['contacts']
         if 'registrant' in contacts and contacts['registrant'] is not None:
             md['Registrant'] = contacts['registrant']
-            standard_ec['Registrant'] = contacts['registrant']
+            standard_ec['Registrant'] = contacts['registrant'].copy()
+            for key, val in contacts['registrant'].items():
+                standard_ec['Registrant'][key.capitalize()] = val
             ec['Registrant'] = contacts['registrant']
+            if 'organization' in contacts['registrant']:
+                standard_ec['Organization'] = contacts['registrant']['organization']
         if 'admin' in contacts and contacts['admin'] is not None:
             md['Administrator'] = contacts['admin']
             ec['Administrator'] = contacts['admin']
-            standard_ec['Admin'] = {key.capitalize(): val for key, val in contacts['admin'].items()}
-            standard_ec['Admin'] = contacts['admin']
+            standard_ec['Admin'] = contacts['admin'].copy()
```

Comment:
Why do we need `copy()` here? is it because of the `key.capitalize()`

Reply:
Yes, otherwise it changes in a few places